MapboxNavigation/NavigationMapView.swift

```diff
@@ -354,24 +363,32 @@ open class NavigationMapView: MGLMapView {
         }
     }
     
+    var activeRouteIndex = 0
+    
+    var routes: [Route]?
+    
     /**
      Adds or updates both the route line and the route line casing
      */
-    public func showRoute(_ route: Route, legIndex: Int? = nil) {
+    public func showRoutes(_ routes: [Route], activeRouteIndex: Int = 0, legIndex: Int = 0) {
```

Comment:
FYI, making `legIndex` non-optional is a breaking change.

Reply:
I'm fine with this change. The way this func was designed before seems incorrect.